<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>994</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SecDialog
{
	background:#2D2D2D;
}</string>
  </property>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>720</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#SaveButton
{
	border:none;
	background:#F33253;
	color:white;
	border-radius:16px;
}

#SaveButton:hover
{
	border:2px solid #F33253;
	background:#2D2D2D;
	color:#F33253;
	border-radius:16px;
}</string>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>  NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_2:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>  AGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_3:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>  DEPARTMENT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_5
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_5:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>   SALARY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_7
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_7:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string> ADDRESS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_8
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#1F1F1F;
}
#label_8:hover
{
	text-align:left;
	padding-left:26px;
	border:none;	
	color:white;	
	background:#333;
}</string>
   </property>
   <property name="text">
    <string>PHONE NUMBER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: blue;
     selection-background-color: darkgray;
 }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_age">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: blue;
     selection-background-color: darkgray;
 }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dep">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: blue;
     selection-background-color: darkgray;
 }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_salary">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: blue;
     selection-background-color: darkgray;
 }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: blue;
     selection-background-color: darkgray;
 }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>620</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: blue;
     selection-background-color: darkgray;
 }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>720</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2
{
	border:none;
	background:#F33253;
	color:white;
	border-radius:16px;
}

#pushButton_2:hover
{
	border:2px solid #F33253;
	background:#2D2D2D;
	color:#F33253;
	border-radius:16px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="addconnection">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>850</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#addconnection
{
	border:none;
	background:#F33253;
	color:white;
	border-radius:16px;
}

#addconnection:hover
{
	border:2px solid #F33253;
	background:#2D2D2D;
	color:#F33253;
	border-radius:16px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
